easybuild/easyconfigs/a/ABINIT/ABINIT-7.6.2-foss-2015a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = "ConfigureMake"
+
+name = 'ABINIT'
+version = '7.6.2'
+
+homepage = 'http://www.abinit.org/'
+description = """Abinit is a plane wave pseudopotential code for doing
+ condensed phase electronic structure calculations using DFT."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = ['http://ftp.abinit.org/']
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = '--enable-mpi --with-mpi-prefix="$EBROOTOPENMPI" --enable-fallbacks '
+configopts += '--with-netcdf-incs="-I$EBROOTNETCDF/include -I$EBROOTNETCDFMINFORTRAN/include" '
+configopts += '--with-netcdf-libs="-L$EBROOTNETCDF/lib64 -lnetcdf -L$EBROOTNETCDFMINFORTRAN/lib -lnetcdff" '
+configopts += '--with-fft-libs="-L$EBROOTFFTW/lib -lfftw3 -lfftw3f" --with-fft-flavor=fftw3 '
+configopts += '--with-trio-flavor=netcdf+etsf_io --with-dft-flavor=libxc --enable-gw-dpc'
+
+patches = ['ABINIT-7.6.2_named-constant.patch',
+           'ABINIT-7.6.2_odamix.patch',
+]
```

Comment:
@AFCJamie please reformat to the below (style, but mainly to avoid hardcoding the version in the patch specs)

```python
patches = [
    'ABINIT-%(version)s_named-constant.patch',
    'ABINIT-%(version)s_odamix.patch',
]
```